### PR TITLE
Driver version updated to 1.1.0

### PR DESCRIPTION
**Pull request by humblec**
Fixes: #353 
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 what are your thoughts on this ? will we do this as the last item for the release or its fine to do it even before it?

**Comment by Madhu-1** [comment]
will do it as the last item

File: pkg/cephfs/driver.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 const (
 
 	// version of ceph driver
-	version = "1.0.0"
+	version = "1.1.0"
```
Comment (Madhu-1) [inline pkg/cephfs/driver.go]: need to update driver version for rbd  also
Note: we may need to do cleanup to have a single variable for the version

**Comment by humblec** [comment]
This PR should be against `v1.1.0` branch. Adding `DNM`.

File: pkg/cephfs/driver.go (hunk 1 above)
Comment (BlaineEXE) [reply pkg/cephfs/driver.go]: You could consider using a method similar to what Rook uses to set the version automatically using `go` linker flags at build time. That would prevent any version updates from being missed due to human error. 

You can look for `LDFLAGS` in Rook's `Makefile` and see how it propagates into `build/makelib/golang.mk`

**Comment by humblec** [comment]
Getting this PR in as part of release process.

**Pull request merged**